README.md

```diff
@@ -51,33 +54,33 @@ On each cluster, using the `skupper` tool, define the Virtual Application Networ
 
    ```bash
    skupper init --site-name public1
-   skupper connection-token public1-token.yaml
+   skupper token create public1-token.yaml
    ```
 2. In the terminal for the second public cluster, deploy the **public2** application router, create a connection token for connections from the **private1** cluser and connect to the **public1** cluster:
 
    ```bash
    skupper init --site-name public2
-   skupper connection-token public2-token.yaml
-   skupper connect public1-token.yaml
+   skupper token create public2-token.yaml
+   skupper link create public1-token.yaml
    ```
 
 3. In the terminal for the private cluster, deploy the **private1** application router and define its connections to the **public1** and **public2** cluster
 
    ```bash
    skupper init --site-name private1
-   skupper connect public1-token.yaml
-   skupper connect public2-token.yaml
+   skupper link create public1-token.yaml
```

Comment:
@nluaces I believe we might hit an issue at this point if the person is using nightly images (or the upcoming 0.7).
That is because token create will generate a token claim that is valid for a single use by default.

Maybe when creating the `public1-token.yaml`, in this example, we should use `--uses 2`. Then I believe it should be ok.

Reply:
Thank you! I will change it.

Reply:
Done!